web.rb

```diff
@@ -6,6 +6,7 @@
 require 'sinatra/base'
 require 'sinatra/respond_with'
 require 'trello'
+require 'trello/core_ext/string'
```

Comment:
In the previous version of the gem, there was an extension called `json_into` that you'd use like `json_into(Trello::Card)`. When I updated the gem, you have to require this specifically to get the same behavior.